setup.py

```diff
@@ -38,6 +38,7 @@ def get_version():
     extras_require={
         "tests": ["pytest", "mmdet", "norfair", "layer"],
         "dev": ["black==21.7b0", "flake8==3.9.2", "isort==5.9.2", "jupyterlab==3.0.14"],
+        "rle": ["pycocotools"]
```

Comment:
we can directly add it to the main requirement.txt file otherwise continuous integration tests wont install this package and wont be able to test your contribution

Reply:
Just be aware that doing that might break the sahi build in windows.

Reply:
latest version of pycocotools works in windows without any issue, we are always using it 👍

Reply:
thanks to this line: https://github.com/ppwwyyxx/cocoapi/blob/fba5219fb0b3e3c390874178c4acf19624da8bb9/PythonAPI/setup.py#L14

Reply:
Oh wow, that was about time! =D

This will make Microsoft Visual C++ 14.0 (or greater) an requirement. But that might be acceptable? 
Just did a clean install of pycocotools and it worked.

Reply:
Pycocotools is now in requirements.txt